lib/l10n/app_bg.arb

```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "помогнете c лапа",
```

Comment:
Let's keep the English app name in all languages.

Reply:
Just Fix that, btw can i get the full strings of bulgarian so i can define all that on the arb so others volunteers just need to use it